examples/albert/arguments.py

```diff
@@ -138,9 +138,14 @@ class AlbertTrainingArguments(TrainingArguments):
     fp16: bool = True
     fp16_opt_level: str = "O2"
     do_train: bool = True
+    do_eval: bool = False
 
+    logging_dir: str = "logs"
+    output_dir: str = "outputs"
     logging_steps: int = 100
+    logging_first_step: bool = True
+    overwrite_output_dir: bool = True
+
     save_total_limit: int = 2
     save_steps: int = 500
-
-    output_dir: str = "outputs"
+    max_steps: int = 10 ** 30
```

Comment:
Is there a point in continuing training after `total_steps` have passed? It seems that with the current schedule, the learning rate for all subsequent updates will be 0

Reply:
it is, but I don't get what do you mean by this

Reply:
What's the point of decoupling these params?

Reply:
Ah, I see the problem.

max_steps is the number of local steps, not global optimizer updates. The reason why previous version was incorrect is because peers would run for 125k local forward-backward cycles which would be much less than 125k scheduler epochs

Reply:
I think it's still unclear that (1) total_steps and max_steps are measured in different units and (2) the difference between them is still elusive for a user.

So I'm for removing max_steps completely (maybe in a future PR since this may require you to test the scripts again)

Reply:
I've added explanations to each parameter to be safe